eth/embobj/plus/comm-v2/icub/EoUpdaterProtocol.c

```diff
@@ -68,7 +68,7 @@
 // - definition (and initialisation) of static variables
 // --------------------------------------------------------------------------------------------------------------------
 
-static const char * s_eouprot_procnames[uprot_proc_numberofthem+1] = {"eLoader", "eUpdater", "eApplication", "eApplPROGupdater", "uprot_proc_None"};
+static const char * s_eouprot_procnames[uprot_proc_numberofthem+1] = {"eLoader", "eUpdater", "eApplication", "eOther", "eApplPROGupdater", "uprot_proc_None"};
```

Comment:
change order because `eApplPROGupdater` will be reverted to be 3.

Reply:
Done.